app/src/org/commcare/utils/DateUtil.kt

```diff
@@ -0,0 +1,37 @@
+package org.commcare.utils
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
+
+@Throws(ParseException::class)
+fun convertIsoDate(inputDate: String, outputPattern: String): String {
```

Comment:
there is already a similar class now in code called `ConnectDateUtils`, can you add these methods over there only ?